packages/kbn-guided-onboarding/src/components/landing_page/guide/__snapshots__/guide_cards.test.tsx.snap

```diff
@@ -0,0 +1,2986 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`guide cards snapshots should render observability cards 1`] = `
+<GuideCards
+  activateGuide={[MockFunction]}
+  activeFilter="observability"
+  cloud={
+    Object {
+      "baseUrl": "base-url",
+      "cloudDefaultPort": "443",
+      "cloudHost": "cloud-host",
+      "cloudId": "mock-cloud-id",
+      "cname": "cname",
+      "deploymentId": "mock-deployment-id",
+      "deploymentUrl": "deployment-url",
+      "elasticsearchUrl": "elasticsearch-url",
+      "isCloudEnabled": true,
+      "isElasticStaffOwned": true,
+      "isServerlessEnabled": false,
+      "kibanaUrl": "kibana-url",
+      "organizationUrl": "organization-url",
+      "profileUrl": "profile-url",
+      "registerCloudService": [MockFunction],
+      "serverless": Object {
+        "projectId": undefined,
+        "projectName": undefined,
+        "projectType": undefined,
+      },
+      "trialEndDate": 2020-10-01T14:13:12.000Z,
+    }
+  }
+  docLinks={
```

Comment:
Hi @rshen91, including this snapshot in this way is causing this test to fail every single time the docLinks object changes, blocking PRs like [this one](https://github.com/elastic/kibana/pull/172301). I doubt that that's desirable. Can we rework this test so that we're not including large objects that are essentially unrelated in the test, and instead make it more focused?